internal/stack/_static/kibana.yml.tmpl

```diff
@@ -4,7 +4,7 @@ server.host: "0.0.0.0"
 {{- $kibana_http2_enabled := fact "kibana_http2_enabled" -}}
 {{- if (and (eq $kibana_http2_enabled "true") (not (semverLessThan $version "8.15.0-SNAPSHOT"))) }}
 server.protocol: http2
-{{ end -}}
+{{- end }}
```

Comment:
This change fixes an error when rendering the kibana.yml file for versions older than 8.15.0-SNAPSHOT

Tested with stack version 8.9.0

```
 $ yq -r ~/.elastic-package/profiles/default/stack/kibana.yml
Error: bad file '/home/mariorodriguez/.elastic-package/profiles/default/stack/kibana.yml': yaml: line 2: mapping values are not allowed in this context
```

This was the rendered file:
```yaml

server.name: kibana
server.host: "0.0.0.0"server.ssl.enabled: true
server.ssl.certificate: "/usr/share/kibana/config/certs/cert.pem"
server.ssl.key: "/usr/share/kibana/config/certs/key.pem"

```
Even that, kibana was able to start successfully.